portfolio/src/main/webapp/script.js

```diff
@@ -80,10 +80,20 @@ function getKdramaRecommendation() {
   document.getElementById("kdrama-title").href = recommendation.link;
 }
 
-async function getComments(){
-    const response = await fetch("/data");
-    const comments = await response.json();
+function getComments(){
+  fetch("/data").then(result => result.json()).then((comments) => {
+    const section = document.getElementById("comments");
+    console.log("try");
+    comments.forEach((comment) => {
+      section.appendChild(createElement(comment, "li"));
```

Comment:
Should we clear the existing children before appending new ones? Currently it will show the same set of comments (and the thank you message) after each click

Reply:
I was thinking to show all the comments that have been posted, not just the new one

Reply:
Yeah, the server should still return all of them not the just the most recent one. Clearing the children list before adding more is just to avoid having something like this 
![iUsVpTHZvYW](https://user-images.githubusercontent.com/2589996/88298039-94492a00-ccf8-11ea-9da4-840195deaabd.png)

Reply:
I seee, okay okay will edit it